### PR TITLE
Issue3

### PR DESCRIPTION
**Pull request by JamieCreek**

### CODE REVIEW

File: bib/east.bib [hunk 1]
```diff
@@ -10121,7 +9984,7 @@ @Book{east:5384
   date       = {0800~},
   title      = {rigs pa'i thigs pa'i don bsdus pa},
   language   = {bo},
-  series     = {snar thang 3724 zhe 115a–116a; sde dge 4233 we 99b5–100b3; co ne we 103a2–103b7; {Peking} 5732 zhe 122b6–123b8},
+  series     = {snar thang 3724 zhe 115a–116a; sde dge 4233 we 99b5-100b3; co ne we 103a2-103b7; {Peking} 5732 zhe 112b6-123b8},
```
Comment (paddymcall) [inline bib/east.bib]: The Peking folio/line number here seems wrong.  A typo?  122b6 seems correct, the text is only one folio long in the other editions.

File: bib/east.bib [hunk 2]
```diff
@@ -10435,11 +10298,11 @@ @article{east:5410
 }
 
 @Book{east:5411,
-  title      = {rtsod pa'i rigs pa'i 'grel pa don rnam par 'byed pa},
   date       = {1100},
-  translator = {{Kumāraśrībhadra} and {'phags pa shes rab} and {'bro seng dkar (śākya 'od)}},
+  title      = {rtsod pa'i rigs pa'i 'grel pa don rnam par 'byed pa},
   language   = {bo},
-  series     = {snar thang 3717 tshe 21b–131b; snar thang 3770 ze 65b–186a; sde dge 4239 zhe 51a3–151a6; co ne zhe 50b3–143b2; {Peking} 5725 tshe 21b2–137a8; {Peking} 5738 ze 71a5–183a7},
+  series     = {snar thang 3717 tshe 21b2–131b7; snar thang 3770 ze 65b4–186a7; sde dge 4239 zhe 51a3–151a6; co ne zhe 50b3–143b2; {Peking} 5725 tshe 21b2–137a8; {Peking}},
```
Comment (paddymcall) [inline bib/east.bib]: {Peking} 5738 ze 71a5–183a7 should not be deleted.

**Review by paddymcall — CHANGES_REQUESTED** [review]
Please correct the changes marked in the comments (two).

File: bib/east.bib (hunk 1 above)
Comment (JamieCreek) [reply bib/east.bib]: Yes, this is simply a typo, I think. The correct folio number is {Peking} 5732 zhe 122b6-123b8 (SUEBS 21.1.T1.).

File: bib/east.bib (hunk 2 above)
Comment (JamieCreek) [reply bib/east.bib]: The second entry in Peking (5738, as you say) was missing from the beginning in EAST, again a typo I think. This corresponds to SUEBS 14.2.T1.
The correct and complete folio numbers thus are: snar thang 3717 tshe 21b2–131b7; snar thang 3770 ze 65b4–186a7; sde dge 4239 zhe 51a3–151a6; co ne zhe 50b3–143b2; {Peking } 5725 tshe 21b2–137a8; {Peking} 5738 ze 71a5–183a7.

**Pull request merged**